.clang-format

```diff
@@ -23,7 +23,6 @@ AlignConsecutiveAssignments: true
 AlignConsecutiveDeclarations: false
 AlignConsecutiveMacros: true
 UseTab: Never
-BreakBeforeBraces: Custom
```

Comment:
You should remove the `Attach` line above for `BreakBeforeBraces` rather than the `Custom` line.